packages/validator-ajv6/src/createAjvInstance.ts

```diff
@@ -42,7 +41,7 @@ export default function createAjvInstance(
   }
 
   // add more custom formats to validate against
-  if (isObject(customFormats)) {
+  if (customFormats instanceof Object) {
```

Comment:
I'm pretty sure this isn't a 1:1 change given that `isObject` is implemented as:
```
function isObject(value) {
  var type = typeof value;
  return value != null && (type == 'object' || type == 'function');
}
```